### PR TITLE
Neopixel LED number config

### LINKED ISSUE #1764: Neopixel header
is there a way to control nonpixel header on TFT screens. it seems that it is in parallel to knob led and it can control only 2 LEDs like the knob. can you increase number of LEDs.

### PR DESCRIPTION
**Pull request by Msq001**
### Description
Now we can set the  number of LED(neopixel)  in the `config.ini` file

### Related Issues
fix #1764

### CODE REVIEW

**Comment by oldman4U** [comment]
Hi. How to connect this to the TFT35 E3 v3.0?

Btw, I believe there is a typo related to this TFT in the config.ini

File: Copy to SD Card root directory to update/config.ini [hunk 1]
```diff
@@ -588,6 +588,13 @@ knob_led_color:0
 #   Options: [true: 1, false: 0]
 knob_led_idle:1
 
+#### Knob LED pixels (only for TFT28 V3.0, TFT35 E3.0, TFT43 V3.0, TFT50 V3.0 & TFT70 V3.0)
+# Set the number of LEDs in the strip connected the "Neopixle" port of TFT.
```
Comment (Guilouz) [inline Copy to SD Card root directory to update/config.ini:592]: "Neopixle" must be "Neopixel"

File: Copy to SD Card root directory to update/config_rrf.ini [hunk 2]
```diff
@@ -588,6 +588,13 @@ knob_led_color:0
 #   Options: [true: 1, false: 0]
 knob_led_idle:1
 
+#### Knob LED pixels (only for TFT28 V3.0, TFT35 E3.0, TFT43 V3.0, TFT50 V3.0 & TFT70 V3.0)
+# Set the number of LEDs in the strip connected the "Neopixle" port of TFT.
```
Comment (Guilouz) [inline Copy to SD Card root directory to update/config_rrf.ini:592]: "Neopixle" must be "Neopixel"

File: TFT/src/User/config.ini [hunk 3]
```diff
@@ -588,6 +588,13 @@ knob_led_color:0
 #   Options: [true: 1, false: 0]
 knob_led_idle:1
 
+#### Knob LED pixels (only for TFT28 V3.0, TFT35 E3.0, TFT43 V3.0, TFT50 V3.0 & TFT70 V3.0)
+# Set the number of LEDs in the strip connected the "Neopixle" port of TFT.
```
Comment (Guilouz) [inline TFT/src/User/config.ini:592]: "Neopixle" must be "Neopixel"

**Review by Guilouz — CHANGES_REQUESTED** [review]

**Comment by oldman4U** [comment]
Ok. So it is time to activate my TFT43 finally👍🏻😁

**Pull request merged**